vega_sim/builders/exceptions.py

```diff
@@ -4,15 +4,15 @@
 from functools import wraps
 
 
-def raise_custom_build_errors(func: Callable):
+def raise_custom_build_errors(func):
     @wraps(func)
-    def wrapped_fn(*args, **kwargs):
+    def wrapper(*args, **kwargs):
         try:
             return func(*args, **kwargs)
         except ValueError as ve:
             raise VegaProtoValueError(ve)
 
-    return wrapped_fn
+    return wrapper
```

Comment:
Type hints weren't working locally for me so this was a copy and paste job from somewhere else where I had type hints working. Turned out to be unrelated so this can be reverted 👍